ttmar_starter.html

```diff
@@ -60,6 +60,45 @@
 
 
   <script>
+    var paragraph1 = document.getElementById("first_paragraph");
+    paragraph1.addEventListener("click", function() {
+      document.body.style.background = "#ccc";
+    })
+
+    var paragraph2 = document.getElementById("second_paragraph");
```

Comment:
This is more a matter of style (and I may have violated this myself in review), but I tend to prefer using `querySelector` or `querySelectorAll` instead of `getElementById` & `getElementsByClassName`.

There's a very slight performance hit, but my reasoning is, by always using selectors, it makes it visually clearer when I'm using an ID vs a class (e.g. `#first_paragraph` looks different than `.first_paragraph`).

Not mandating that you take this approach, just something to think about!